modules/cbProcessInfo/bpmpopup.php

```diff
@@ -34,20 +34,30 @@
 			'pflowid' => $_REQUEST['pflowid'],
 			'bpmrecord' => $_REQUEST['bpmrecord'],
 		]);
+		$MapObject = new cbMap();
+		$Mapid = $check->fields['fieldmap'];
+		$refrenceField = '';
+		if ($check->fields['fieldmap']) {
+			$MapObject->id = $Mapid;
+			$MapObject->mode = '';
+			$MapObject->retrieve_entity_info($Mapid, "cbMap");
+			$xml = simplexml_load_string($MapObject->column_fields['content']);
+			$refrenceField = (string)$xml->linkfields->originfield;
```

Comment:
it is the target field we are looking for here. Change that, the rest LGTM

Reply:
ok